_includes/templates/download.html

```diff
@@ -88,10 +88,12 @@ <h2>{{ page.latestversion }} {{CURRENT_RELEASE}} <a type="application/rss+xml" h
       <div>
         <img src="/assets/images/os/med_linux.png" alt="linux">
         <span>
-          <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.ppc64le }}.tar.gz" class="dl">PPC64 Linux</a>
+          <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.ppc64 }}.tar.gz" class="dl">PPC64 Linux</a>
+<!--
             <span><a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.ppc64 }}.tar.gz" class="dl" id="lin64ppc">64 bit</a>
             -
             <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.ppc64le }}.tar.gz" class="dl" id="lin64ppcle">64 bit LE</a></span>
```

Comment:
It seems like htmlproofer is not ignoring the commented-out elements, so perhaps best to just remove these lines entirely? We can always git revert later.

Reply:
Yeah, deleted them and added a comment instead.